src/style/rating.css

```diff
@@ -0,0 +1,7 @@
+.rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+
+  margin-bottom: 24px;
```

Comment:
margins for BEM-block?
![image](https://user-images.githubusercontent.com/75496546/221808858-64117cf4-e952-4ec9-89c9-132bb22788ec.png)